### PR TITLE
Kreivo de Paseo

### PR DESCRIPTION
**Pull request by pandres95**

### CODE REVIEW

**Review by olanod — COMMENTED** [review]
Should we remove any mention of Rococo?

**Comment by pandres95** [comment]
We use rococo locally since zombienet with Kusama-local is broken. Let me check if we can fix that part.

**Comment by pandres95** [comment]
Done (finally)!

File: node/src/chain_spec/kreivo/live.rs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+#[cfg(not(feature = "paseo"))]
+pub fn chain_spec() -> ChainSpec {
+	ChainSpec::from_json_bytes(include_bytes!("./kreivo_kusama_chainspec.json").as_slice()).unwrap()
+}
+
+#[cfg(feature = "paseo")]
+use sp_core::crypto::Ss58Codec;
+
+#[cfg(feature = "paseo")]
+pub fn chain_spec() -> ChainSpec {
```
Comment (olanod) [inline node/src/chain_spec/kreivo/live.rs:12]: Since the chain is also live we can do the same and include the json bytes.

**Pull request merged**